Dockerfile.s390x

```diff
@@ -4,11 +4,14 @@ LABEL maintainer "LoZ Open Source Ecosystem (https://www.ibm.com/developerworks/
 
 ADD licenses/ /licenses
 ADD LICENSE /licenses/
+RUN mkdir -p /opt/cni/bin
+
+FROM scratch
+
+COPY --from=base /licenses /licenses
```

Comment:
Do we need this because it is a different license for s390?

Reply:
Good catch. We don't. I will update this

Reply:
It was there. I just moved around but left where it was.

FROM s390x/debian:9.8-slim

...
ADD licenses/ /licenses
ADD LICENSE /licenses/

...